File: levelledmobs-plugin-bukkit/src/main/java/io/github/arcaneplugins/levelledmobs/bukkit/logic/group/Group.java
package io.github.arcaneplugins.levelledmobs.bukkit.logic.group;

import java.util.LinkedList;
import java.util.List;
import org.jetbrains.annotations.NotNull;

public final class Group {

    /* vars */

    private final String identifier;
    private final List<String> items = new LinkedList<>();

    /* constructors */

    public Group(final @NotNull String identifier) {
        this.identifier = identifier;
    }

    public Group(
        final @NotNull String identifier,
        final @NotNull List<String> items
    ) {
        this.identifier = identifier;
        this.items.addAll(items);
    }

    /* getters and setters */

    @NotNull
    public String getIdentifier() { return identifier; }

    @NotNull
    public List<String> getItems() { return items; }

}

File: levelledmobs-plugin-bukkit/src/main/kotlin/io/github/arcaneplugins/levelledmobs/bukkit/logic/group/Group.kt
package io.github.arcaneplugins.levelledmobs.bukkit.logic.group

class Group {
    constructor(identifier: String){
        this.identifier = identifier
    }

    constructor(identifier: String, items: MutableList<String>){
        this.identifier = identifier
        this.items.addAll(items)
    }

    val identifier: String
    val items: MutableList<String> = mutableListOf()
}